com.gitlab.idevecore.Pomodoro.json

```diff
@@ -0,0 +1,39 @@
+{
+  "app-id": "com.gitlab.idevecore.Pomodoro",
```

Comment:
it should be `io.gitlab`.

see https://docs.flatpak.org/en/latest/conventions.html#application-ids